### PR TITLE
Provider gui config from packages

### PR DESCRIPTION
**Pull request by HughParry**

### CODE REVIEW

File: provider-gui/services/contract/config.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        defaultEnvironment: environment,
+        defaultNetwork: network,
+        account: {
+            password: '',
+            address: 'currentAccount',
+        },
+        database: {
+            development: { type: DatabaseTypes.enum.mongoMemory, endpoint: '', dbname: 'prosopo_test', authSource: '' },
+        },
+        server: {
```
Comment (forgetso) [inline provider-gui/services/contract/config.ts]: This is for the image server. You shouldn't need to declare it.

File: provider-gui/services/contract/config.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            baseURL: 'http://localhost',
+            port: 9229,
+            fileServePaths: '[]',
+        },
+        networks: {
```
Comment (forgetso) [inline provider-gui/services/contract/config.ts]: This should default to sensible values if you don't supply it.

https://github.com/prosopo/captcha/blob/d5b42d16c2a10aec21a8d8b57a211a1f8e0dde07/packages/types/src/networks/index.ts#L10-L40

**Review by forgetso — CHANGES_REQUESTED** [review]
Couple of very minor points.

File: provider-gui/services/contract/config.ts (hunk 1 above)
Comment (HughParry) [reply provider-gui/services/contract/config.ts]: Removed

File: provider-gui/services/contract/config.ts (hunk 2 above)
Comment (HughParry) [reply provider-gui/services/contract/config.ts]: Sorted

**Review by forgetso — APPROVED** [review]

**Pull request merged**